haskell/test/Lorentz/Contracts/Test/FA2.hs

```diff
@@ -607,9 +586,57 @@ fa2Spec fa2Originate = do
             let isOperatorQuery = mkView (#operator .! operatorParam) consumer
             lCallEP fa2contract (Call @"Is_operator") isOperatorQuery
 
-            (validate . Right $
-              lExpectViewConsumerStorage consumer
-                [(#operator .! operatorParam, #is_operator .! True)])
+            lExpectViewConsumerStorage consumer
+                [(#operator .! operatorParam, #is_operator .! True)]
+
+  -- Check whether "update operator", "remove operator" operations are executed only by contract owner.
```

Comment:
Are these tests new? Why did you add them as part of morley update?

Reply:
Well, this is weir how this appeared in management tests and got merged onto master. I'll make a separate commit for it

Reply:
And where is this promised separate commit? :pled: